pinot-core/src/test/java/com/linkedin/pinot/core/io/util/PinotDataBitSetTest.java

```diff
@@ -27,6 +27,25 @@
   private static final Random RANDOM = new Random();
   private static final int NUM_ITERATIONS = 1000;
 
+  @Test
+  public void testGetNumBitsPerValue() {
+    for (int i = 0; i < NUM_ITERATIONS; i++) {
+      int cardinality = RANDOM.nextInt(Integer.MAX_VALUE) + 1;
+      int actual = PinotDataBitSet.getNumBitsPerValue(cardinality);
+      if (cardinality == 1) {
```

Comment:
The probability of random picking `cardinatlity = 1` is too low. Maybe it's better to test this separately outside of this loop since it's an edge case?